modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/Modal.js

```diff
@@ -241,7 +247,16 @@ const Modal = ({
 							>
 								{url && (
 									<>
-										{loading && <ClayLoadingIndicator />}
+										{loading ? (
+											<ClayLoadingIndicator />
+										) : (
+											<span
+												className="sr-only"
+												role="status"
+											>
+												{Liferay.Language.get('loaded')}
+											</span>
+										)}
```

Comment:
Shouldn't this change be in Clay, in the [ClayLoadingIndicator component](https://github.com/liferay/clay/blob/master/packages/clay-loading-indicator/src/index.tsx)? There are many places in DXP besides modal where we use component in this way. /cc @matuzalemsteles

Reply:
I'm not sure, because this state from what I'm understanding should appear after the loading is done. I think this should work more as an announcer and removed after a few seconds.

Reply:
As @matuzalemsteles says, it's a message that should appear when the ClayLoadingIndicator finishes. Based on Matu's comments, I added that the message will be deleted after 1 second.

Reply:
@matuzalemsteles Would it make sense to add to Clay something like `<ClayLoadingStatusMessage>`?

Then, a common pattern in DXP would be
```js
{loading ? (
	<ClayLoadingIndicator />
) : (
	<ClayLoadingStatusMessage />

	<MyContent />
)}
```

Reply:
I'd prefer use a LiveAnnouncer-like implementation that would already take care of that, we've implemented that in Clay https://github.com/liferay/clay/blob/master/packages/clay-core/src/live-announcer/LiveAnnouncer.tsx, but it is still not 100% reliable because we need to treat it as a singleton so we will still refactor this implementation to not be in React but only using JS to avoid some bugs from SRs also due to the renderings that exist. I think we could keep it here for now and then do it using the LiveAnnouncer API which we can make public soon.

Reply:
@matuzalemsteles So the Clay `LiveAnnouncer`, once it's ready, will make the solution in this PR unnecessary? Or will it require some code on DXP side to make it work? In either case, we can put this PR on hold, or, if the delivery is urgent, merge as a temporary solution and revert later.

Reply:
How urgent is urgent? :slightly_smiling_face: I'm assuming couple of days is acceptable. If LiveAnnouncer will not likely be available for weeks or more, we can merge it for sure.

Reply:
For now we are not going to work again on LiveAnnouncer to redo it in pure JS because we have other accessibility priorities for BofA, but I should come back to that in the next few weeks. You won't need to implement anything on the DXP side so that's good.

I would go ahead with this solution and later on we can move this to use the LiveAnnouncer API.

Reply:
Sounds good!